aws-sns-topic/src/test/java/software/amazon/sns/topic/DeleteHandlerTest.java

```diff
@@ -62,17 +62,6 @@ public void handleRequest_SimpleSuccess() {
                 .attributes(attributes)
                 .build();
         when(proxyClient.client().getTopicAttributes(any(GetTopicAttributesRequest.class))).thenReturn(getTopicAttributesResponse);
-        final ListSubscriptionsByTopicResponse listSubscriptionsByTopicResponse = ListSubscriptionsByTopicResponse.builder()
-                .subscriptions(software.amazon.awssdk.services.sns.model.Subscription.builder()
-                        .subscriptionArn("arn:aws:sns:us-east-1:123456789012:sns-topic-name:27735157-80a9-4735-9427-090465a162d2")
-                        .endpoint("abc@xyz.com")
```

Comment:
Why do we use this "abc@xyz.com" as endpoint?

Reply:
this is email protocol: the endpoint is an email address. https://docs.aws.amazon.com/sns/latest/dg/sns-email-notifications.html

Reply:
This was a mimic that we have an email subscription with endpoint as `abc@xyz.com` as the subscription for this topic.

In original implementation, we need to get the subscriptions for this topic and unsubscribe, while in this PR, we removed such steps.